src/components/MyAccount/RequestsTab/UpdateLocation.tsx

```diff
@@ -166,7 +170,6 @@ const UpdateLocation = ({
       </Box>
     ),
     onClose: () => {
-      setModalProps(confirmLocationChangeModalProps as DefaultModalProps)
```

Comment:
You no longer need to do this because it resets on the next rerender, right?

Reply:
correct!